### PR TITLE
Addind accouting (new method) posts for september

### PR DESCRIPTION
**Pull request by jfoucry**
New post for september accouning

### CODE REVIEW

**Comment by jfoucry** [comment]
We change the accounting files as we change accounting year, like we did last year but this time we did it on the website at the same time we did on the accounting software. We can archive the old pages to be more clear in the website.

Btw. I added the headers

File: content/post/comptabilite-2019-2020.en.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Accounting 2019-2020"
+date: 2019-10.03
```
Comment (simpnu) [inline content/post/comptabilite-2019-2020.en.md]: The date format is incorrect, should be ` date: 2019-10-03`

File: content/post/comptabilite-2019-2020.fr.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Comptabilité 2019-2020"
+date: 2019-10-03
+draft: false
+---
+Comptes au `2019/09`:
```
Comment (simpnu) [inline content/post/comptabilite-2019-2020.fr.md]: Missing space before the ":"

**Comment by simpnu** [comment]
> We change the accounting files as we change accounting year, like we did last year but this time we did it on the website at the same time we did on the accounting software. We can archive the old pages to be more clear in the website.

Ok thanks for the info! :)
I think we can keep the old pages, maybe we could rename them to clarify this?

**Pull request merged**